TestLib/XCTestCase/XCUIApplication+GREYEnvironment.m

```diff
@@ -23,7 +23,7 @@ - (void)grey_configureApplicationForLaunch {
   NSString *insertionKey = @"DYLD_INSERT_LIBRARIES";
   NSString *insertionValue = @"@executable_path/Frameworks/AppFramework.framework/AppFramework";
   NSString *alreadyExistingValue = [mutableEnv valueForKey:insertionKey];
-  if (alreadyExistingValue) {
+  if (alreadyExistingValue && ![alreadyExistingValue containsString:insertionValue]) {
     insertionValue = [NSString stringWithFormat:@"%@,%@", alreadyExistingValue, insertionValue];
```

Comment:
Change the comma to a colon. This should be colon separated.

Reply:
The colon was the key point I wasn't aware of. Thats for pointing. I have updated it.